### PR TITLE
Update docs after migrating from apm integration package to ES apm-data plugin in 8.15

### PR DESCRIPTION
**Pull request by carsonip**
## Description

Remove mentions of integration packages as apm-server moved away from it in 8.15

### Documentation sets edited in this PR

_Check all that apply._

- [x] Stateful (`docs/en/observability/*`)
- [ ] Serverless (`docs/en/serverless/*`)
- [ ] Integrations Developer Guide (`docs/en/integrations/*`)
- [ ] None of the above

### Related issue
Closes # 

## Checklist

- [x] Product/Engineering Review
- [ ] Writer Review

### Follow-up tasks

_Select one._

* This PR does _not_ need to be ported to another doc set because:
  - [ ] The concepts in this PR only apply to one doc set (serverless _or_ stateful)
  - [ ] The PR contains edits to both doc sets (serverless _and_ stateful)
* This PR needs to be ported to another doc set:
  - [ ] Port to stateful docs: \<link to PR or tracking issue>
  - [ ] Port to serverless docs: \<link to PR or tracking issue>

### CODE REVIEW

File: docs/en/observability/apm/ingest-pipelines.asciidoc [hunk 1]
```diff
@@ -65,6 +65,5 @@ If you prefer more guidance, see one of these tutorials:
 
 * <<apm-filters-ingest-pipeline>> — Learn how to obfuscate passwords stored in the `http.request.body.original` field.
 * <<apm-data-stream-rerouting>> — Learn how to reroute APM data to user-defined APM data streams.
-* {fleet-guide}/data-streams-pipeline-tutorial.html[Transform data with custom ingest pipelines] — A basic Elastic integration tutorial where you learn how to add a custom field to incoming data.
```
Comment (carsonip) [inline docs/en/observability/apm/ingest-pipelines.asciidoc:68]: This is a shame, as the tutorial is good, but some instructions do not apply to ES plugin

File: docs/en/observability/apm/data-streams.asciidoc [hunk 2]
```diff
@@ -109,7 +109,7 @@ For example, consider traces that would originally be indexed to `traces-apm-def
 ]
 ----
 
-For more about other ingest pipelines called by default, e.g. `traces-apm@custom`, see {fleet-guide}/data-streams.html#data-streams-pipelines[integration data streams ingest pipelines].
+For more about other ingest pipelines called by default, e.g. `traces-apm@custom`, see definition of `\*-apm*@default-pipeline` ingest pipelines from {es} apm-data plugin.
```
Comment (colleenmcginnis) [inline docs/en/observability/apm/data-streams.asciidoc:112]: Should this link somewhere? Usually when we say "see {something}" the {something} is a linked resource.

**Review by colleenmcginnis — COMMENTED** [review]
One small question.

File: docs/en/observability/apm/data-streams.asciidoc (hunk 2 above)
Comment (carsonip) [reply docs/en/observability/apm/data-streams.asciidoc:112]: There are no docs around it, not like integration data streams ingest pipelines. What I was trying to convey was that users should just go to ES and check out the relevant pipelines themselves. I don't feel comfortable linking to the [actual source code](https://github.com/elastic/elasticsearch/tree/main/x-pack/plugin/apm-data), as pinning source code version to doc version is not feasible. Any good idea how to rewrite this?

File: docs/en/observability/apm/ingest-pipelines.asciidoc (hunk 1 above)
Comment (colleenmcginnis) [reply docs/en/observability/apm/ingest-pipelines.asciidoc:68]: Tracking next steps for that tutorial in https://github.com/elastic/observability-docs/issues/4314.

File: docs/en/observability/apm/data-streams.asciidoc (hunk 2 above)
Comment (colleenmcginnis) [reply docs/en/observability/apm/data-streams.asciidoc:112]: >go to ES and check out the relevant pipelines themselves

Ok. This makes sense to me. Where would they go or what would they do to find the relevant pipelines? Stack management in Kibana? Using an API? Something else?

Comment (carsonip) [reply docs/en/observability/apm/data-streams.asciidoc:112]: I'm thinking about stack management, but if a power user likes API, they can do so too.

Comment (colleenmcginnis) [reply docs/en/observability/apm/data-streams.asciidoc:112]: >I'm thinking about stack management, but if a power user likes API, they can do so too.

We can keep it simple with just Stack management.

What about something like this? Feel free to adjust as needed.

```suggestion
To find other ingest pipelines from the {es} apm-data plugin that are called by default, go to {kibana-ref}/management.html[*Stack management*] → *Ingest pipelines* and search for `apm`.
Default APM ingest pipelines will follow the pattern `*-apm*@default-pipeline`.
```

Comment (carsonip) [reply docs/en/observability/apm/data-streams.asciidoc:112]: Looks perfect, thanks!

File: docs/en/observability/apm/data-streams.asciidoc [hunk 3]
```diff
@@ -109,7 +109,8 @@ For example, consider traces that would originally be indexed to `traces-apm-def
 ]
 ----
 
-For more about other ingest pipelines called by default, e.g. `traces-apm@custom`, see {fleet-guide}/data-streams.html#data-streams-pipelines[integration data streams ingest pipelines].
+To find other ingest pipelines from the {es} apm-data plugin that are called by default, go to *Stack management* → *Ingest pipelines* {kibana-ref}/management.html[[in Kibana] and search for `apm`.
```
Comment (colleenmcginnis) [inline docs/en/observability/apm/data-streams.asciidoc:112]: My mistake! 

```suggestion
To find other ingest pipelines from the {es} apm-data plugin that are called by default, go to *Stack management* → *Ingest pipelines* {kibana-ref}/management.html[in Kibana] and search for `apm`.
```

**Review by colleenmcginnis — APPROVED** [review]

**Review by mdbirnstiehl — APPROVED** [review]

**Pull request merged**